de.fin_ger.OvGUCanteen.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "de.fin_ger.OvGUCanteen",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "gnome-ovgu-canteen",
+    "desktop-file-name-suffix": " (Flatpak)",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=network",
+        "--env=GSETTINGS_BACKEND=keyfile"
```

Comment:
```suggestion
```
Already the default since 19.08

Reply:
I tested it without this environment variable and it didn't work. Glib always chose the dconf gsettings backend. So I guess it's needed.

Reply:
I don't think it is correct. It works fine here with `flatpak run --env=GSETTINGS_BACKEND='' io.github.fin_ger.OvGUCanteen`.

Reply:
indeed, it is working with the flathub built version. However, it failed when building locally. I will double check and built a version without the environment variable locally.

Reply:
I checked again and now it is working. I don't know why it failed before... :(